static/style.css

```diff
@@ -126,26 +130,24 @@ a:hover {
 
 .ColoredBackground{
```

Comment:
Considering the stylizations below, I believe that the class name is not so appropriate. What do you think about giving this class a more semantic name? Some suggestions are: Box, ContainerWrapper, DemoWrapper, ExempleWrapper